src/basics.php

```diff
@@ -155,3 +155,16 @@ function loadPHPUnitAliases()
         require_once dirname(__DIR__) . DS . 'tests' . DS . 'phpunit_aliases.php';
     }
 }
+
+if (!function_exists('get_var_type')) {
```

Comment:
Since when are we using underscore as convention for method names? getVarType() would be the consistent naming IMO.

Reply:
The "var" sounds odd here... getTypeOrClassName() maybe?